presto-docs/src/main/sphinx/getting-started-original.rst

```diff
@@ -0,0 +1,56 @@
+===============
```

Comment:
Please ignore this file. It is a copy of the original. Changes are in getting-started.rst.